.github/actions/zenodo-upload/zenodo_upload.py

```
@@ -0,0 +1,101 @@
+
+    # Load metadata from file or use default
+    if metadata_file and os.path.exists(metadata_file):
+        with open(metadata_file, "r") as f:
+            if metadata_file.endswith(".cff"):
+                cff_data = yaml.safe_load(f)
+                metadata = {
+                    "metadata": {
+                        "title": cff_data["title"],
+                        "upload_type": "software",
+                        "description": cff_data["abstract"],
+                        "creators": [{"name": f"{a['given-names']} {a['family-names']}"} for a in cff_data["authors"]],
```

Ruff (0.8.2)

Line too long (119 > 99)

(E501)